include/irr/asset/IAssetManager.h

```diff
@@ -434,11 +434,15 @@ class IAssetManager : public core::IReferenceCounted
         /** Keeping assets around (by their pointers) helps a lot by letting the loaders retrieve them from the cache 
 		and not load cpu objects which have been loaded, converted to gpu resources and then would have been disposed of.
 		However each dummy object needs to have a GPU object associated with it in yet-another-cache for use when we convert CPU objects to GPU objects.*/
-        void convertAssetToEmptyCacheHandle(IAsset* _asset, core::smart_refctd_ptr<core::IReferenceCounted>&& _gpuObject, uint32_t referenceLevelsBelowToConvert=~0u)
+        void convertAssetToEmptyCacheHandle(IAsset* _asset, core::smart_refctd_ptr<core::IReferenceCounted>&& _gpuObject, uint32_t referenceLevelsBelowToConvert=0u)
```

Comment:
keep my default of `~0u` we want default not to "leak memory"

Reply:
very important

Reply:
we might have to talk about it. Generally it may result in assets converted to empty cache handles that arent actually cache handles and whats more you cannot make GPU objects from them any more because they've dropped needed resources

Reply:
as with discord you will change to ~0u